test/readMPAS.f90

```diff
@@ -8,13 +8,29 @@ subroutine assert(condition,message)
   endif
 end subroutine
 
+
+
 module readMPAS
     use :: polympo
     use iso_c_binding
     implicit none
     integer, parameter :: MPAS_RKIND = selected_real_kind(12)
     
 contains
+
+function epsilonDiff(a,b) result(isSame)
+  implicit none
+  real(kind=MPAS_RKIND) :: a,b,delta
+  parameter (delta=1.0e-8)
+  logical :: isSame
+  !delta=1.0e-8
```

Comment:
remove this line (commented out)

Reply:
fixed with https://github.com/SCOREC/polyMPO/commit/3ad1bfe776e24416282bbd4a16bc8a5cb4503ec2